CPS-?/README.md

```diff
@@ -0,0 +1,229 @@
+- Operational security: The group should be aware of their surroundings and take steps to minimize the risk of physical threats, such as surveillance or theft.
+
+4. Develop a contingency plan for the group in the event of a security breach. This plan should include steps to revoke compromised keys, notify other members of the group of the breach, and initiate recovery procedures. Experts may need to provide guidance to Delegator Representatives. 
+
+5. Implement measures to protect the group's identities, such as using pseudonyms or code names when communicating with each other. Consider the Constitutional Committee or Delegator Representatives may need confirmation from the Emergency Group for actions taken.
+
+6. Limit the number of people who know the identities of the Emergency Group members to minimize the risk of information leaks. The identity of the group members should only be disclosed on a need-to-know basis.
+
+7. When Stake Pool Operators are requred to take action, have trusted experts communicate with them via the normal Stake Pool Operator channels so they know they are getting information from the correct source. Switch to less public channels if the normal channels are suspected of being compromised.
+
+8. Regularly review and update the security measures in place to ensure that they remain effective. Possibly do a periodic practice disaster recovery drill on one of the test networks.
+
+### #4: Operational Security (OPSEC) of the committee.
+  * Physical security
+  * Digital security
+
+**Recommendation:**
+
+### #5: Committee keys lost or stolen or committee keys sold to a bad actor.
+
+**Recommendation:**
+
+### #6: DRep keys lost or stolen or DRep keys sold to a bad actor.
+
+**Recommendation:**
+
+### #7: Lack of governance for an extended period of time.
+  * Uncontrolled hard forks by SPOs
+  * Risk of no confidence
+
+**Recommendation:**
+
+### #8: Lack of trust in voter tooling.
+
+**Recommendation:**
+
+### #9: Trust in the wallets.
+  * wallet voting attack vector
+  * hardware wallet support
+
+**Recommendation:**
+
+
+### #10: Corruption - in any form, any number of actors.
+  * Code of Conduct mitigation
+  * on-chain mitigation
+
+
+**Recommendation:**
+
+### #11: Collusion - the bad version of collaboration to do something illegal or undesirable.
+
+**Recommendation:**
+
+### #12: Vote buying - Ada kickbacks.
+
+**Recommendation:**
+
+### #13: IGCO - initial governance coin offering
+  * offering voters a token of unknown value by a representative to acquire delegators
+  * including tokens other than Ada
+
+**Recommendation:**
+
+### #14: Powerful Voter Collusion
+  * demotivates smaller bag holders
+  * cause a loss of participation
+  * too many voters abstain
+  * minority controls the outcome
+  * can push through any proposal
+
+**Recommendation:**
+
```

Comment:
Has to be addressed in the constitution.  Just like it is in the US constitution: https://constitution.congress.gov/browse/essay/intro-2-2-2/ALDE_00000031/ (I know nothing of this at any kind of depth but the idea is sound)